homeassistant/components/backup/http.py

```diff
@@ -58,12 +65,25 @@ async def get(
         headers = {
             CONTENT_DISPOSITION: f"attachment; filename={slugify(backup.name)}.tar"
         }
+        reader: IO[bytes]
         if agent_id in manager.local_backup_agents:
             local_agent = manager.local_backup_agents[agent_id]
             path = local_agent.get_backup_path(backup_id)
-            return FileResponse(path=path.as_posix(), headers=headers)
-
-        stream = await agent.async_download_backup(backup_id)
+            if not password:
+                return FileResponse(path=path.as_posix(), headers=headers)
+            try:
+                reader = await hass.async_add_executor_job(open, path.as_posix(), "rb")
```

Comment:
Where is the reader closed?

Reply:
I must have forgotten that, well spotted!